src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -618,8 +627,17 @@ def process_dump(
         include_extras=include_extras,
         overwrite=overwrite,
         flat=flat,
+        also_remote=also_remote,
     )
 
+    if also_remote:
+        echo.echo_report(
+            '`--also-remote` set to True. Will try to retrieve additional files from the `workdir` of the remote Computer.'  # noqa: E501
```

Comment:
not really the workdir of the Computer (which is the base directory of the scratch) but rather of the calcjob itself